src/axom/quest/IntersectionShaper.hpp

```diff
@@ -43,30 +46,6 @@
     #include "conduit_blueprint_mcarray.hpp"
   #endif
 
-// clang-format off
```

Comment:
@gunney1 and others -- I moved this block to `shaping_helpers` since it's needed for sample-based shaping as well.
I `ifdef`d out the MFEM dependency there, so `IntersectionShaper` does not require MFEM.